wiki/Contests/en.md

```diff
@@ -1,97 +1,165 @@
-| 9           | [Furries in a Blender - Storm World](https://osu.ppy.sh/beatmapsets/319473)                            | ![osu!taiko][o!t]    | [Firce777](https://osu.ppy.sh/users/274072)         | [Results](https://osu.ppy.sh/home/news/2015-06-27-monthly-beatmapping-contest-9-results) | [Score #9](https://osu.ppy.sh/p/contestresults?c=18)   |
-| 10          | [Hylian Lemon - Foresight Is for Losers](https://osu.ppy.sh/beatmapsets/342751)                        | ![osu!catch][o!c]    | [ZiRoX](https://osu.ppy.sh/users/200768)            | [Results](https://osu.ppy.sh/home/news/2015-08-06-monthly-beatmap-contest-10-results-ctb) | [Score #10](https://osu.ppy.sh/p/contestresults?c=19)  |
-| 11          | [Freedom Planet - Dragon Valley (Toni Leys Remix feat. Esteban Bellucci)](https://osu.ppy.sh/beatmapsets/385056) | ![osu!mania][o!m]    | [LordRaika](https://osu.ppy.sh/users/3265023)       | [Results](https://osu.ppy.sh/home/news/2015-12-03-monthly-beatmapping-contest-11-results) | [Score #11](https://osu.ppy.sh/p/contestresults?c=20)  |
-| 12          | [IAHN - Candy Luv](https://osu.ppy.sh/beatmapsets/456054)                                              | ![osu!standard][o!s] | [Taeyang](https://osu.ppy.sh/users/2732340)         | [Results](https://osu.ppy.sh/home/news/2016-05-26-monthly-beatmapping-contest-12-results) | [Score #12](https://osu.ppy.sh/p/contestresults?c=21)  |
-| 13          | [BilliumMoto - HDHR](https://osu.ppy.sh/beatmapsets/569888)                                            | ![osu!standard][o!s] | [RyoKazuka](https://osu.ppy.sh/users/6258586)       | [Results](https://osu.ppy.sh/home/news/2017-04-02-monthly-beatmapping-contest-13-results)  | [Score #13](https://docs.google.com/spreadsheets/d/1CysSPvMCrkace40s4aGnpIdmc-TNEJZ72QGKz0MPBqc) |
-| March 2017  | [nanobii - HYPER★DRIVE](https://osu.ppy.sh/beatmapsets/639991)                                         | ![osu!catch][o!c]    | [Ascendance](https://osu.ppy.sh/users/2931883)      | TBA                                             | [Score #14](https://osu.ppy.sh/community/contests/44)      |
-| April 2017  | [cYsmix - Breeze](https://osu.ppy.sh/beatmapsets/629575)                                               | ![osu!mania][o!m]    | [Soul Evans](https://osu.ppy.sh/users/4490770)      | TBA                                             | [Score #15](https://osu.ppy.sh/community/contests/46)  |
-
-### Aspire Contest
-
-Aspire Contests are beatmapping contests with one special exception, they are not bound to the [Ranking Criteria](/wiki/Ranking_Criteria).
+- First time in the series expand to cover osu!taiko, osu!catch, and osu!mania (Top 10 only).
+- First time in the series to also cover linkless beatmaps.
```

Comment:
I'd just remove all of the notes for these since it's obvious when you view the spreadsheets /etc

also idek what this one means